pkgs/native_assets_cli/tool/rename.dart

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
```

Comment:
Is that some helper script you use? I don't think we need this to be checked into the repo.